### DISCUSSION TITLE
Not able to install R4 package

### DISCUSSION THREAD

**Discussion by itsnitishgoyal** [nazrulworld/fhir.resources#130] [General]
Not able to install R4 package- is 4.3.0 is available ?

pip install fhir.resources==4.3.0

**Comment by nazrulworld** [comment]
I think fhir.resources never had 4.3.0 release. If you want to use R4, I would suggest https://pypi.org/project/fhir.resources/6.4.0/

**Reply by itsnitishgoyal** [reply]
Ok Narjul Thank you, will use 6.4.0.

**Reply by itsnitishgoyal** [reply]
@nazrulworld  where i can see the list of supported resources . Is group resource and coverage supported? Also can you share any link for using FHIR path , any example to extract the element value using FHIR path? 
Thanks in Advance
